source/agora/common/Serializer.d

```diff
@@ -151,3 +158,65 @@ public void serializePart (scope const(ubyte)[] record, scope SerializeDg dg)
     serializePart(record.length, dg);
     dg(record);
 }
+
+/*******************************************************************************
+
+    Unsigned Integers to Serialized variable length integer
+    and return it as a ubyte[].
+
+    VarInt Size
+    size <= 253(0xFC)  -- 1 byte   ubyte
+    size <= USHORT_MAX -- 3 bytes  (0xFD + ushort)
+    size <= UINT_MAX   -- 5 bytes  (0xFE + uint)
+    size <= ULONG_MAX  -- 9 bytes  (0xFF + ulong)
+
+    Params:
+        T = Type of struct to serialize
+        var = Instance of `T` to serialize
+
+    Returns:
+        The serialized convert variable length integer
+
+*******************************************************************************/
+
+public ubyte[] toVarInt (T) (scope const auto ref T var)
```

Comment:
There is no need to use `scope const auto ref` here, it's just an integer, passing by value should be fine.

You can replace with `const T var`.

Reply:
Applied the suggestion.